code/datums/elements/food/processable.dm

```diff
@@ -32,3 +33,12 @@
 
 	mutable_recipes += list(list(TOOL_PROCESSING_RESULT = result_atom_type, TOOL_PROCESSING_AMOUNT = amount_created, TOOL_PROCESSING_TIME = time_to_process))
 	return COMPONENT_NO_AFTERATTACK
+
+///So people know what the frick they're doing without reading from a wiki page (I mean they will inevitably but i'm trying to help, ok?)
+/datum/element/processable/proc/OnExamine(atom/A, mob/user, list/examine_list)
+	SIGNAL_HANDLER
+
+	if(amount_created > 1)
+		examine_list += "<span class='notice'>It can be turned into [amount_created] [initial(result_atom_type.name)]s with a <b>[tool_behaviour]</b>!</span>"
+	else
+		examine_list += "<span class='notice'>It can be turned into a [initial(result_atom_type.name)] with a <b>[tool_behaviour]</b>!</span>"
```

Comment:
```suggestion
		examine_list += "<span class='notice'>It can be turned into [amount_created] [initial(result_atom_type.name)]s with \a <b>[tool_behaviour]</b>!</span>"
	else
		examine_list += "<span class='notice'>It can be turned into \a [initial(result_atom_type.name)] with \a <b>[tool_behaviour]</b>!</span>"
```

Also, these names aren't meant to be displayed. They're enums. For example, `TOOL_ROLLINGPIN` is "rollingpin", not "rolling pin". WIRECUTTER is "wirecutter", when its best read as "a pair of wirecutters".

Make a separate helper proc to create displayed text.